format-specs/schema.json

```diff
@@ -11,95 +12,84 @@
-              ],
-              "description": "JSON object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of OGC:CRS84."
-            },
-            "edges": {
-              "type": "string",
-              "enum": ["planar", "spherical"],
-              "description": "Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'."
-            },
-            "orientation": {
-              "type": "string",
-              "enum": ["counterclockwise"],
-              "description": "Winding order of exterior ring of polygons; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order."
-            },
-            "bbox": {
-              "type": "array",
-              "description": "Bounding Box of the geometries in the file, formatted according to RFC 7946, section 5.",
-              "items": [
-                {
-                  "type": "number",
-                  "description": "The westmost constant longitude line that bounds the rectangle (xmin)."
-                },
-                {
-                  "type": "number",
-                  "description": "The minimum constant latitude line that bounds the rectangle (ymin)."
-                },
-                {
-                  "type": "number",
-                  "description": "The eastmost constant longitude line that bounds the rectangle (xmax)."
+      "description": "Metadata about geometry columns. Each key is the name of a geometry column in the table.",
+      "additionalProperties": {
+        "type": "object",
+        "required": ["encoding", "geometry_type"],
+        "properties": {
+          "encoding": {
+            "type": "string",
+            "const": "WKB",
+            "description": "Name of the geometry encoding format. Currently only 'WKB' is supported."
+          },
+          "geometry_type": {
+            "oneOf": [
+              {
+                "type": "string",
+                "const": "Unknown"
+              },
+              {
+                "$ref": "#/definitions/geometry_type"
+              },
+              {
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/geometry_type"
                 },
-                {
-                  "type": "number",
-                  "description": "The maximum constant latitude line that bounds the rectangle (ymax)."
-                }
-              ]
+                "uniqueItems": true
+              }
+            ],
+            "description": "The geometry type(s) of all geometries, or 'Unknown' if they are not known."
+          },
+          "crs": {
+            "$ref": "https://proj.org/schemas/v0.5/projjson.schema.json"
```

Comment:
Removed the `null` schema for CRS, becuase null is not mentioned in the spec.

Reply:
I think we may have had a discussion about this a while ago, and it seems like we had a preference for having a literal `null` (like we require a literal `Unknown` for geometry type) instead of having the key be missing.

Reply:
Okay, I can change it back and update the spec if that's confirmed.

Reply:
Yes, having

Reply:
Yes, having `"crs": null` is different from a missing "crs" key

It is mentioned here in the spec:

https://github.com/opengeospatial/geoparquet/blob/811d52bdb57814dbf0c13049481b8f3b2367b31d/format-specs/geoparquet.md?plain=1#L85